app/payment-request/get-payment-request.js

```diff
@@ -25,8 +25,11 @@ const getPaymentRequest = async (categoryId = 1) => {
       'received',
       'receivedFormatted',
       'ledger',
+      'marketingYear',
+      'daysWaiting',
       'netValue'
-    ]
+    ],
+    order: [['received']]
```

Comment:
Can you add tests in `test/integration/local/payment-request/get-payment-request.test.js` to ensure this ordering is correctly displayed, both ordering and new fields?

Reply:
added test